testproject/build.gradle

```diff
@@ -35,11 +41,16 @@ neoForge {
         server {
             server()
         }
+        apitest {
```

Comment:
Doesn't this add all of the `testproject` mod to the run despite the dependency on only `api`?

Reply:
Because it adds the entire `testproject` mod, which contains all of the source sets.